endpoints-support.md

```diff
@@ -0,0 +1,92 @@
+***Legend***
+
+| Item | Meaning |
+| ------------------ | --------------------------------------------- |
+|  :white_check_mark: | Endpoint implemented in the Puppet SDK for this API version :tada: |
+|  :heavy_multiplication_x: | Endpoint considered as 'out-of-scope' for the Puppet SDK |
+|  :heavy_minus_sign: | Endpoint not available for this API Version |
+
+<br />
+
+***Notes***
+
+* If an endpoint is not marked as implemented for a specific API, it can still be used in compatibility mode for the supported API versions.
+* If an example is not working on a supported API version, verify the [HPE OneView REST API Documentation](http://h17007.www1.hpe.com/docs/enterprise/servers/oneview4.1/cic-api/en/index.html#)  for the API version being used, since the expected attributes for that resource might have changed.
+<br />
+
+## HPE OneView
+| Endpoints                                                                       | Verb     | V200 | V300 | V500 |V600 |V800 | V1000 | V1200 |
+| --------------------------------------------------------------------------------------- | -------- | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: |
```

Comment:
We dont need to include versions 200-600. Remove them.

Reply:
This is still there in ruby endpoint support. that's why I haven't removed them.